src/firefly/js/charts/ui/ChartsTableViewPanel.jsx

```diff
@@ -452,7 +452,8 @@ class ChartsPanel extends React.Component {
         if (optionsShown) {
             return (
                 <div className='ChartPanelOptions'>
-                    <OptionsWrapper  {...{chartId, tableModel, tblStatsData, tblPlotData, tblHistogramData, chartType}}/>
+                    <OptionsWrapper toggleOptions={this.toggleOptions}
+                        {...{chartId, tableModel, tblStatsData, tblPlotData, tblHistogramData, chartType}}/>
```

Comment:
I am wondering if toggleOptions can be handled here. 
Otherwise we need to propagate this prop to all options components (for now it included HistogramOptions) and handle situations when toggleOptions do not make sense, like in Charts dropdown.